.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Please be sure to add JS linters to your project. Follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) to do so 😉

Reply:
I already added the Eslint and test that there is no errors in my local environment but I don't know why it's not working with GitHub action ( you can see that the .eslintrc.json appears in the new added files)

Reply:
Yes, the .eslintrc.json is added but there is code mission in your [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml). This file is the one responsible for handling the Github actions and right now you don't have ESlint code there. As you can see in the previous link, there is a section of code regarding ESlint action so please be sure to add it to your file 😉